docs/guide/file_management.md

```diff
@@ -100,6 +100,17 @@ It also accepts optional query parameters:
 
 The [ez\_render\_field](twig_functions_reference.md#ez_render_field) Twig helper will by default generate a working link.
 
+### Download link generation
+
+To generate a direct download link for the `File` Field Type you have to create
+a Route Reference with the `ez_route` helper, passing `content` and `File` Field identifier as parameters.
+Optional parameter `inLanguage` may be used to specify File content translation.
+
+```twig
+  {% set routeReference = ez_route( 'ez_content_download', {'content': content, 'fieldIdentifier': 'file', 'inLanguage': content.prioritizedFieldLanguageCode  } ) %}
```

Comment:
`prioritizedFieldLanguageCode` property is marked as internal and shouldn't be used in the user space code.  Ref. https://github.com/ezsystems/ezpublish-kernel/blob/368f3cfbdebf61a887be87a3cd0b0e7bf1f9f676/eZ/Publish/Core/Repository/Values/Content/Content.php#L39-L48